### DISCUSSION TITLE
Issue in PE+ 13

### DISCUSSION THREAD

**Discussion by sanjeevstunner** [ROM-builders/temporary#22822] [General]
https://cirrus-ci.com/task/4737905822793728?logs=build#L298
https://cirrus-ci.com/task/6024078205648896?logs=build#L224
https://cirrus-ci.com/task/5208258294579200?logs=build#L320

I tired multiple times and the same error exists 

I tired to remove the tree using the local manifest remove project attribute, but it threw non-existent project error
You can see that here
https://cirrus-ci.com/task/6078712689459200?logs=tsync#L2

**Comment by Apon77** [comment]
you can't remove project which does not exist in the manifest! anyway i cleaned, hope things will be fixed

**Reply by sanjeevstunner** [reply]
Thanks. Have a good day
